test/srt/models/lora/test_lora.py

```diff
@@ -77,11 +85,19 @@ def inference(self, prompts, lora_set, tp_size, torch_dtype, max_new_tokens):
         print("=================== testing inference =======================")
         base_path = lora_set["base"]
         all_lora_paths = lora_set["loras"]
-        batch_lora_paths = [None]
-        i = 0
-        for _ in range(len(prompts) - 1):
-            batch_lora_paths.append(all_lora_paths[i])
-            i = (i + 1) % len(all_lora_paths)
+        batch_lora_paths = []
+        for prompt_id in range(len(prompts)):
+            if prompt_id < len(all_lora_paths):
+                batch_lora_paths.append(all_lora_paths[prompt_id])
+            else:
+                batch_lora_paths.append(all_lora_paths[0])
+        # batch_lora_paths = [None]
+        # i = 0
+        # for _ in range(len(prompts) - 1):
+        #     batch_lora_paths.append(all_lora_paths[i])
+        #     i = (i + 1) % len(all_lora_paths)
+
+        print("batch_lora_paths", batch_lora_paths)
```

Comment:
Why not use the original way, which will test for mixing None and real adapters?